test/users/engin/refComparison.skipif

```diff
@@ -0,0 +1 @@
+CHPL_COMM!=none
```

Comment:
Oh boy :( You are right, sorry about the confusion. Do you want me to fix it or remove it altogether?